core/templates/pages/footer_js_libs.html

```diff
@@ -15,3 +9,9 @@
     <script src="/webpack_bundles/<%= htmlWebpackPlugin.files.js[chunk] %>"></script>
   <% } %>
 <% } %>
+
+<% if (webpackConfig.mode == 'production') { %>
+  <script src="/third_party/generated/js/third_party.min.js"></script>
+<% } else { %>
+  <script src="/third_party/generated/js/third_party.js"></script>
+<% } %>
```

Comment:
Why is this moved?

Reply:
Third party libs need angular. Angular is included in bundle so, we have to load `third_party.js` after our webpack bundles.